admin/config-ui/class-configuration-page.php

```diff
@@ -104,8 +104,7 @@ public function show_wizard() {
 				printf(
 					/* translators: %s expands to Yoast SEO. */
 					__( '%s &rsaquo; Configuration Wizard', 'wordpress-seo' ),
-					'Yoast SEO'
-				);
+					'Yoast SEO' );
```

Comment:
Ok, checked. This has to do with the _going in and out of PHP_ and a different (better) solution will actually start to be enforced in WPCS as of v 0.12.0.

The original code looked like:
```php
			<title><?php
				printf(
					/* translators: %s expands to Yoast SEO. */
					__( '%s &rsaquo; Configuration Wizard', 'wordpress-seo' ),
					'Yoast SEO'
				);
			?></title>
```

Changing it to the below will sort it properly:
```php
			<title>
				<?php
				printf(
					/* translators: %s expands to Yoast SEO. */
					__( '%s &rsaquo; Configuration Wizard', 'wordpress-seo' ),
					'Yoast SEO'
				);
				?>
			</title>
```

I suspect the same will apply to the other instances where this was happening, but will have a look still anyway.